keyboards/ymdk/np24/u4rgb6/u4rgb6.h

```diff
@@ -36,18 +36,32 @@
     { k05, k15, k25, k35 }  \
 }
 
-#define LAYOUT_ortho_4x6( \
+#define LAYOUT_ortho_4x6_right( \
     k00, k01, k02, k03, k04, k05, \
     k10, k11, k12, k13, k14, k15, \
     k20, k21, k22, k23, k24, k25, \
     k30, k31, k32, k33, k34, k35  \
 ) { \
-    { k00, k10, k20, k30 }, \
-    { k01, k11, k21, k31 }, \
-    { k02, k12, k22, k32 }, \
-    { k03, k13, k23, k33 }, \
+    { k05, k15, k25, k35 }, \
     { k04, k14, k24, k34 }, \
-    { k05, k15, k25, k35 }  \
+    { k03, k13, k23, k33 }, \
+    { k02, k12, k22, k32 }, \
+    { k01, k11, k21, k31 }, \
+    { k00, k10, k20, k30 }  \
+}
+
+#define LAYOUT_ortho_4x6_left( \
+    k00, k01, k02, k03, k04, k05, \
+    k10, k11, k12, k13, k14, k15, \
+    k20, k21, k22, k23, k24, k25, \
+    k30, k31, k32, k33, k34, k35  \
+) { \
+    { k30, k20, k10, k00 }, \
+    { k31, k21, k11, k01 }, \
+    { k32, k22, k12, k02 }, \
+    { k33, k23, k13, k03 }, \
+    { k34, k24, k14, k04 }, \
+    { k35, k25, k15, k05 }  \
 }
 
```

Comment:
```suggestion

#define LAYOUT_ortho_4x6 LAYOUT_ortho_4x6_right
```

Reply:
I don't understand this. Any reading I should do?
I couldn't find ortho_4x6 in the layouts folder either. Am I missing something?

Reply:
The issue is that you're changing `LAYOUT_ortho_4x6` to `LAYOUT_ortho_4x6_right`, and you have `LAYOUTS = ortho_4x6` in your rules.mk.  

So either that rules.mk declaration needs to go, or you need to have `LAYOUT_ortho_4x6` mapped to the "proper" layout, so that the layout info is all correct.

Reply:
I actually have ortho_4x6_right and ortho_4x6_left in case people want the connector on either end of the device.
What should I be doing in that case?

Reply:
That's fine.  The issue is this line:
https://github.com/trenthoh/qmk_firmware/blob/48518d75d88cb3cd8f1be5816432d3fb2091f981/keyboards/ymdk/np24/u4rgb6/rules.mk#L24

This expects a layout macro that matches.  Specifically, `LAYOUT_ortho_4x6`.  

Your change removes the layout of that name, which is causing the issue (check the lint check below).  You either need to remove the `ortho_4x6` from `LAYOUTS`, or you need to add some sort of alias, so that the API is aware of an actual layout.  

The best option is the alias, which is why I recommended it.  The simpler option is removing `ortho_4x6` from `LAYOUTS.

Reply:
#define LAYOUT_ortho_4x6 LAYOUT_ortho_4x6_right
#define LAYOUT_ortho_4x6 LAYOUT_ortho_4x6_left
So I do this?

Reply:
Yeah, but the `#define LAYOUT_ortho_4x6 LAYOUT_ortho_4x6_left` needs to go after `LAYOUT_ortho_4x6_left` define

Reply:
Didn't get past lint either way. Removed ortho_4x6 from the layouts list.
Should it be in the layouts/community folder?

Reply:
Oh, shoot, you're right! 

In that case, a default config would need to be added to /layouts/default, or the layout removed in the `LAYOUTS` define.

Reply:
Removed from layouts to get it outta my hair first. I'll look into it again later.